packages/common/tests/test-utils.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const test = require('ava');
+
+const { omit } = require('../util');
+
+test('omit removes a key from object', (t) => {
```

Comment:
Is there a reason we're not using https://underscorejs.org/#omit ?

Reply:
It's slated for removal in v5.0.0 https://github.com/lodash/lodash/wiki/Roadmap 
(originally I had it with _omit, then replaced it and then did this, see convo with me and lauren https://github.com/nasa/cumulus/pull/789#discussion_r254258493)